aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -291,6 +294,9 @@ open class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknown
 
     private var focusOnVisible = true
 
+    var inputConnection: InputConnection? = null
+    var inputConnectionEditorInfo: EditorInfo? = null
```

Comment:
I wonder if we should keep a `WeakReference` to those objects. These are objects used deeply withing the Android editing subsystem and I'm not sure if we'll introduce other bugs if we keep strong references to them. WDYT?

Reply:
good call, updated the `inputConnection` to use a `WeakReference` in d20fb8572a8f29886d51dbf4335c46f33944e10a.

The `var inputConnectionEditorInfo` is an actual new instance that copies the attributes values [by value on this line](https://github.com/wordpress-mobile/AztecEditor-Android/pull/834/files#diff-7ca71cd1f009ab8438e0901205001b14R684) so, left as it was 👍